### DISCUSSION TITLE
New Shared Libraries

### DISCUSSION THREAD

**Discussion by khargy** [icloud-photos-downloader/icloud_photos_downloader#491] [Q&A] [answered]
Seems like it's only downloading from my "personal" library and not the new "shared" library. Is that supported yet? Is there a way to switch it?

**Comment by ebattistini** [comment, marked as answer]
It does not look like it is working at the moment. You need to turn it off if you want to download the whole library

**Reply by khargy** [reply]
Ok, thanks... although best I can tell there's no simple way to "turn it off" if you delete your shared library it has to copy the whole thing back over to your personal library. Anyway this feature just came out so I have no problem waiting for it to be supported just wanted to make sure I wasn't missing something.
